tensorflow/lite/kernels/slice.cc

```diff
@@ -117,7 +117,7 @@ TfLiteStatus Prepare(TfLiteContext* context, TfLiteNode* node) {
                  begin->type == kTfLiteInt32 || begin->type == kTfLiteInt64);
   TF_LITE_ENSURE(context,
                  size->type == kTfLiteInt32 || size->type == kTfLiteInt64);
-  TF_LITE_ENSURE(context, NumDimensions(begin) == NumDimensions(size) == 1);
+  TF_LITE_ENSURE(context, (NumDimensions(begin) == NumDimensions(size)) == 1);
```

Comment:
I don't think this is right... this is checking that `NumDimensions(begin) == NumDimensions(size)`, and then I think that the resulting boolean == 1, which is a no-op. I'm guessing this is supposed to check that both begin and size have 1 dimension, which needs two checks.

Reply:
@skye thanks for pointing this out, i have fixed the warning as well as taken care of your comments, kindly review and approve the PR.